source/php/Component/Footer/Partials/subfooter.blade.php

```diff
@@ -0,0 +1,41 @@
+@php
+$directionClass = $baseClass . '__subfooter--' . ($subfooter['direction'] === 'vertical' ? 'vertical' : 'horizontal');
```

Comment:
isn't this redundant? if you're trying to ensure it's not null wouldn't this be better? but even then shouldn't this be fixed in the components php file?
```suggestion
$directionClass = $baseClass . '__subfooter--' . ($subfooter['direction'] ?? 'horizontal');
```